Sources/ResilientDecoding/ResilientDictionary.swift

```diff
@@ -20,37 +20,37 @@ extension KeyedDecodingContainer {
   /**
    Decodes a `Resilient` dictionary, omitting elements as errors are encountered.
    */
-  public func decode<Element>(_ type: Resilient<[String: Element]>.Type, forKey key: Key) throws -> Resilient<[String: Element]>
+  public func decode<Value>(_ type: Resilient<[String: Value]>.Type, forKey key: Key) throws -> Resilient<[String: Value]>
```

Comment:
Curious why you moved from `Element` to `Value`

Reply:
The generic parameter for `Dictionary` is `Value`, not `Element`. Arrays have `Element`s